src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py

```diff
@@ -97,6 +97,12 @@ def test_hostcfgd(self, test_name, test_data, fs):
         fs.create_dir(hostcfgd.FeatureHandler.SYSTEMD_SYSTEM_DIR)
         MockConfigDb.set_config_db(test_data["config_db"])
         with mock.patch("hostcfgd.subprocess") as mocked_subprocess:
+
```

Comment:
Can you remove this extra blank line?

Reply:
It would be fine if we want to merge this PR to get the fix faster. Removing this empty line with anther PR is fine by me.

Reply:
@yxieca extra line removed, please check now.